src/internal/m365/service/groups/backup.go

```diff
@@ -67,9 +67,14 @@ func ProduceBackupCollections(
 			break
 		}
 
+		catStr := scope.Category().PathType().String()
+		if scope.Category().PathType() == path.ChannelMessagesCategory {
+			catStr = "messages"
```

Comment:
Should we maybe update the stringer value? I don't see why that has to be `channelMessages` to begin with.

Reply:
I was thinking of doing a follow-up for this.